### ISSUE TITLE
Support for ISO-variations

### ISSUE THREAD

**Issue by cupofjoakim** [crsnbrt/keysim#10]
We ISO-users are, without a doubt, a minority. It's understandable if the project doesn't want to support other layouts than ANSI, but as a swede I'd really love support for ISO.

It might open uncomfortable doors however, as some might request AZERTY support and stuff like that.

**Comment by crsnbrt** [comment]
I only included the 60% iso for now because I didn't want the layout dropdown to have an ansi and iso option for every size. I'd prefer it to be a secondary toggle option instead. 

But yea tbh I'm not sure the best way to handle all the Latin ISO variations without becoming too overwhelming.

**Comment by TJScalzo** [comment]
It may be outside the current scope of the project, but adding support for importing from [Keyboard Layout Editor](http://www.keyboard-layout-editor.com/#/) ([GitHub repo](https://github.com/ijprest/keyboard-layout-editor)) might be a good solution to this issue. It's a robust platform that seems to be pretty well known in the custom keycap community. Being able to take any layout from KLE and interact with it in this simulator would be a great feature to have.

**Comment by crsnbrt** [comment]
@TJScalzo I agree it would be a nice feature. The biggest hurdle to that is I'll have to rewrite a lot of the renderer to work with this json structure. not a huge deal but I already went down the path of using the qmk format so it's slightly more than I want to tackle atm. I'll leave this issue open as I may come back to this.

**Comment by escaparrac** [comment]
Maybe you can just add ISO and JIS to the most common ones; 60, 65, tkl and full, a really big % of the population would be satisfied with that.

**Comment by KaKi87** [comment]
Hi,

+1 for AZERTY

Thanks
